source/features/cross-deleted-pr-branches.tsx

```diff
@@ -0,0 +1,62 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+
+function inPR(): void {
+	let deletedBranch: string | undefined;
+	const lastBranchAction = select.all(`
+		.discussion-item-head_ref_deleted .commit-ref,
+		.discussion-item-head_ref_restored .commit-ref
+    `).pop();
+
+	if (lastBranchAction && lastBranchAction.closest('.discussion-item-head_ref_deleted')) {
+		deletedBranch = lastBranchAction.textContent!.trim();
+	}
+
+	// Making deleted branch link to the default fork
+	for (const el of select.all('.commit-ref[title].head-ref[title]')) {
+		const [repo] = el.title.split(':', 1);
+		const branchName = el.textContent!.trim();
+		if (branchName !== 'unknown repository') {
+			if (branchName === deletedBranch) {
```

Comment:
These two conditions can be merged
```suggestion
			if (branchName === deletedBranch) {
```

Reply:
done in new commit.